app/models/decidim/problems/problem.rb

```diff
@@ -18,6 +18,9 @@ class Problem < Problems::ApplicationRecord
       component_manifest_name "problems"
 
       belongs_to :challenge, foreign_key: "decidim_challenges_challenge_id", class_name: "Decidim::Challenges::Challenge"
+      has_many :solutions,
```

Comment:
This 'restrict` should be checked in the command otherwise the user will have a crash